app/src/README.md

```diff
@@ -1,17 +1,29 @@
-Node/Grunt modules for backend’s frontend workflow.
+# Bolts backend’s frontend workflow using grunt
```

Comment:
Ah, btw, this readme is only there out of one reason: npm complains if it is missing :-)